src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const GROUP_STYLE = sourceString.trim().split(';');
+  const GET_CLEAN_STYLE = GROUP_STYLE.map((pair) => pair.trim()).filter(
+    (style) => style !== '',
+  );
```

Comment:
The filter method creates a new array with all elements that pass the test implemented by the provided function. There is no need to call map with a trim before filter as the filter can handle the trimming and checking for an empty string in one iteration.